supply/lib/supply/setup.rb

```diff
@@ -43,45 +42,46 @@ def store_metadata(listing)
     def download_images(listing)
       UI.message("🖼️  Downloading images (#{listing.language})")
 
-      # We cannot download existing screenshots as they are compressed
-      # But we can at least download the images
       require 'net/http'
 
-      IMAGES_TYPES.each do |image_type|
-        if ['featureGraphic'].include?(image_type)
-          # we don't get all files in full resolution :(
-          UI.message("📵  Due to a limitation of the Google Play API, there is no way for `supply` to download your existing feature graphic. Please copy your feature graphic to `metadata/android/#{listing.language}/images/featureGraphic.png`")
-          next
-        end
+      allowed_imagetypes = [Supply::IMAGES_TYPES, Supply::SCREENSHOT_TYPES].flatten
 
+      allowed_imagetypes.each do |image_type|
         begin
           UI.message("Downloading `#{image_type}` for #{listing.language}...")
 
-          url = client.fetch_images(image_type: image_type, language: listing.language).last
-          next unless url
+          urls = client.fetch_images(image_type: image_type, language: listing.language)
+          next if urls.nil? || urls.empty?
+
+          image_counter = 1 # Used to prefix the downloaded files, so order is preserved.
+          urls.each do |url|
+            url = url.gsub(url.match("=.*").to_s, "") # Remove everything after '=' (if present)
+            url = url.gsub("-rw", "") # Remove "-rw" (if present) so the url will become either 'jpg' or 'png' (https://github.com/fastlane/fastlane/pull/14322#issuecomment-473017130)
```

Comment:
We don't need this any more if we removed everything after and including `=`, right?

Reply:
https://github.com/fastlane/fastlane/pull/14322/commits/f720f66544def11d165bd8d02fbef66a8f73a429